cms/static/sass/elements/_forms.scss

```diff
@@ -172,8 +172,97 @@ form {
   }
 }
 
+// +Forms - License selector UI
+// ====================
+.license-img {
+  padding: 4px;
+}
+
+ul.license-types {
+  text-align: middle;
+  vertical-align: middle;
+  display: inline-block;
+
+  li {
+    display: inline-block;
+  }
+
+  .action.license-button {
+    @include grey-button;
+    @extend %t-action2;
+
```

Comment:
NIT: remove spaces between rules linked to the same level / selector. (keep space between these rules and any deeper selectors. )